.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
+
```

Comment:
Maybe we can add a section on changing the code of conduct?  like

"Code of conduct changes will be submitted openly as a PR to this document, and must be approved by ....."

Reply:
I could go either way.  Would you mind either suggesting specific language or where to put it, or handle this as a follow-on?